src/components/WidgetPropertiesForm/index.tsx

```diff
@@ -72,23 +80,60 @@ export default function WidgetPropertiesForm({ id, onClose }: WidgetPropertiesFo
     },
   });
 
```

Comment:
You should move the defineMessages outside the component. Currently you are defining messages at each render.